credentials/src/main/kotlin/web5/sdk/credentials/model/PresentationSubmission.kt

```diff
@@ -0,0 +1,29 @@
+package web5.sdk.credentials.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Represents a presentation submission object.
+ *
+ * @see [Presentation Submission](https://identity.foundation/presentation-exchange/spec/v2.0.0/#presentation-submission)
+ */
+public data class PresentationSubmission(
```

Comment:
See below for motivation
```suggestion
public class PresentationSubmission(
```